bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_BaseObject.cpp

```diff
@@ -51,8 +51,10 @@
 #include <sofa/core/collision/Pipeline.h>
 #include <sofa/core/collision/Intersection.h>
 #include <sofa/core/objectmodel/ConfigurationSetting.h>
-
 #include <sofa/core/ExecParams.h>
+#include <sofa/core/CategoryLibrary.h>
+#include <pybind11/stl.h>
```

Comment:
Because I use the getCategories method of the CategoryLibrary class on line 140 it must be included otherwise I will have an error
and the pybind11 is necessary for the binding I think

Reply:
I will check on my side and correct it if necessary then I will push

Reply:
@VannesteFelix  it 's done !

Reply:
Hi @VannesteFelix 

I noticed that I could no longer compile after the modifications I made because before I had done git fetch --all and git merge in all my branches.
I know it's not my edits that's causing the problem. To check this I did a git reset --HARD to return to a previous state that was working normally before recompiling but the problem persists. But I still submitted the changes I had made.


I tried to look but I don't see what is causing my compliation to fail. Can you check with me? Please

Reply:
Beware, the pybind11/stl is needed as this is the way to get auto-conversion from c++ stl container to their corresponding python version. I'm not sure this missing conversion cause compilation failure but more a runtime one.

Reply:
Ok my bad @damienmarchal, sorry about that @samFabrice.